lib/project_types/node/messages/messages.rb

```diff
@@ -119,10 +119,12 @@ module Messages
           extended_help: <<~HELP,
             {{bold:Options:}}
               {{cyan:--host=HOST}}: Bypass running tunnel and use custom host. HOST must be HTTPS url.
+              {{cyan:--port=PORT}}: Use custom port.
```

Comment:
cc @MeredithCastile, considering this is scoped under `serve`, is it obvious the port refers to the tunnel?